OpenRA.Mods.Common/UtilityCommands/ConvertPngToShpCommand.cs

```diff
@@ -35,7 +35,21 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 		{
 			var inputFiles = GlobArgs(args).OrderBy(a => a).ToList();
 			var dest = inputFiles[0].Split('-').First() + ".shp";
-			var frames = inputFiles.Select(a => PngLoader.Load(a));
+
+			var frames = inputFiles.Select(a =>
```

Comment:
The original code for this command was being very silly.  This method can simplify down to
```csharp
[Desc("PNGFILE [PNGFILE ...]", "Combine a list of PNG images into a SHP")]
void IUtilityCommand.Run(Utility utility, string[] args)
{
	var inputFiles = GlobArgs(args).OrderBy(a => a).ToList();
	var dest = inputFiles[0].Split('-').First() + ".shp";

	var frames = inputFiles.Select(a => new Png(File.OpenRead(a))).ToList();
	var size = new Size(frames[0].Width, frames[0].Height);
	if (frames.Any(f => f.Width != size.Width || f.Height != size.Height))
		throw new InvalidOperationException("All frames must be the same size");

	using (var destStream = File.Create(dest))
		ShpTDSprite.Write(destStream, size, frames.Select(f => f.Data));

	Console.WriteLine(dest + " saved.");
}
```
and `ToBytes` deleted completely.